webpack/components/extensions/HostDetails/Tabs/RemoteExecutionActions.js

```diff
@@ -0,0 +1,35 @@
+import { API_OPERATIONS, post } from 'foremanReact/redux/API';
+import REX_JOB_INVOCATIONS_KEY from './RemoteExecutionConstants';
+import { foremanApi } from '../../../../services/api';
+import { getResponseErrorMsgs } from '../../../../utils/helpers';
+import { renderTaskStartedToast } from '../../../../scenes/Tasks/helpers';
+
+const errorToast = (error) => {
+  const message = getResponseErrorMsgs(error.response);
+  return message;
+};
+
+const jobInvocationParams = ({ hostname, packageName }) => ({
+  job_invocation: {
+    feature: 'katello_package_install',
+    targeting_type: 'Static Query',
+    inputs: {
+      package: packageName,
+    },
+    search_query: `name ^ (${hostname})`,
```

Comment:
Searching by the host id might be faster / more precise?

Reply:
When you select "via customized remote execution" on the old page, this is the search that's filled out for you.  I feel like it's better for the user to see the hostname there as opposed to, say, `id = 12247`.  So that's why I did it this way.  For the immediate action here ("via remote execution") I suppose that's not as important, but it would at least be consistent.

Reply:
i didn't realize the user sees this. all good then